.circleci/config.yml

```diff
@@ -70,7 +70,7 @@ workflows:
           filters:
             tags:
               # Matches e.g. "release_123"
-              only: /^release_[0-9]+$/
+              only: /^DEPRECATED_release_[0-9]+$/
```

Comment:
Can't quite get rid of this file without getting errors from the test step. We can unhook it from the repo after the we're sure GH actions works on the release.

Reply:
I'll make a followup jira item for it after this is merged.

Reply:
https://jira.unity3d.com/browse/MLA-1442 for followup